client/my-sites/post-type-list/post-action-counts/index.jsx

```diff
@@ -0,0 +1,101 @@
+/* @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import QueryPostStats from 'components/data/query-post-stats';
+import { getNormalizedPost } from 'state/posts/selectors';
+import { getPostStat } from 'state/stats/posts/selectors';
+
+class PostActionCounts extends PureComponent {
+	static propTypes = {
+		globalId: PropTypes.string,
+	};
+
+	renderCommentCount() {
+		const { commentCount: count, numberFormat, translate } = this.props;
+
+		return (
+			<li>
+				{ translate(
+					'%(count)s Comment',
+					'%(count)s Comments',
+					{
+						count,
+						args: { count: numberFormat( count ) },
+					}
+				) }
+			</li>
+		);
+	}
+
+	renderLikeCount() {
+		const { likeCount: count, numberFormat, translate } = this.props;
+
+		return (
+			<li>
+				{ translate(
+					'%(count)s Like',
+					'%(count)s Likes',
+					{
+						count,
+						args: { count: numberFormat( count ) },
+					}
+				) }
+			</li>
+		);
+	}
+
+	renderViewCount() {
+		const { viewCount: count, numberFormat, translate } = this.props;
+
+		return (
+			<li>
+				{ translate(
+					'%(count)s View',
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( '%(count)s Total View', '%(count)s Total Views' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).